docs/organise/help/communication.md

```diff
@@ -1,5 +1,11 @@
 ## Conferences
 
+### Hackathons UK unconference 
+
+This is a conference for hackathon organisers of UK hackathons to get together, help eachother, network and discuss various topics. No fixed adgenda.
```

Comment:
- `each other` should be two words
- `agenda` only has one `d` in it

Otherwise all good 😊